deployment/components/common/worker-mounts-combined.yaml

```
@@ -38,6 +41,9 @@
   - name: host-usr-lib
     mountPath: "/host-usr/lib"
     readOnly: true
+  - name: host-lib
+    hostPath:
+      path: "/lib"
```

The field in volumeMounts is `mountPath`. Also, you need to mount it with the `host-` prefix


```suggestion
    mountPath: "/host-lib"
    readOnly: true
```

...and ditto volumemounts you added into other files